src/presentation/controller/common/LimitedPartnerController.ts

```diff
@@ -263,14 +264,33 @@ class LimitedPartnerController extends AbstractController {
 
           await this.conditionalPreviousUrl(ids, pageRouting, request, tokens);
 
+          const isCeaseDatePage: boolean =
```

Comment:
Might want to consider a private function called by sendData or if buildLimitedPartnerErrorRenderData were in the abstract class, we could do something along the lines of what has been done for the create function below.

Reply:
updated again, moved isCeaseDatePage into pageType module

Reply:
should be ready now once sonar has run